src/util.ts

```diff
@@ -5,12 +5,12 @@ import isNode from "detect-node";
 // tslint:disable-next-line:no-var-requires
 const querystring = isNode ? require("querystring") : require("querystring-browser");
 
-interface IOriginalRequestMetadata {
-  [key: string]: any;
-  requestHeaders: any;
-  requestHost: string;
-  requestMethod: string;
-  requestPath: string;
+export interface IPersistableData {
+  lang?: string;
```

Comment:
Is the `[key: string]: any` not necessary anymore with this (i.e. does it automatically change `{ lang: "x", requestPath: "y" }` to `IPersistableData`?